qd_proto.lua

```diff
@@ -39,6 +41,21 @@ function qd_proto.prefs_changed()
     end
 end
 
+-- Parsing QD message.
+-- @param proto Protocol object.
+-- @param type QD message type.
+-- @param tvb_buf Input buffer.
+-- @param packet_info Packet information.
+-- @param subtree Subtree for display fields in Wireshark
+--                (for concrete type message).
+local function parsing_message(proto, type, tvb_buf, packet_info, subtree)
```

Comment:
Usually, the names of functions that implement some actions begin with a verb in the imperative mood. Should be `parse_message`.